tests/conftest.py

```diff
@@ -69,12 +82,29 @@ def start_process(path, command):
         return process
 
 
+@pytest.fixture(name="get_experiment_json")
+def fixture_get_experiment_json(request):
+    experiment_slug = request.config.getoption("--experiment-slug")
+    slug_server = request.config.getoption("--experiment-server")
+
+    match slug_server:
+        case "prod":
+            url = (
+                f"https://experimenter.services.mozilla.com/api/v6/experiments/{experiment_slug}/"
+            )
+        case "stage":
+            url = f"https://stage.experimenter.nonprod.dataops.mozgcp.net/api/v6/experiments/{experiment_slug}/"  # noqa: E501
+
+    return requests.get(url).json()
```

Comment:
I'm guessing this implodes if I accidentally set an `--experiment-server=production` (instead of "prod"). Not saying it's worth fixing, but in the off chance we want some extra checks or throw errors on invalid data.